odc/geo/_xr_interop.py

```diff
@@ -768,6 +822,8 @@ def to_rgba(
     ) -> xarray.DataArray:
         return to_rgba(self._xx, bands=bands, vmin=vmin, vmax=vmax)
 
+    crop = _wrap_op(crop)
```

Comment:
I didn't know whether to add this to just the "ODC extension base class" or both the "ODCExtensionDa" and "ODCExtensionDs". I went with the second option, but can easily change to the first one

Reply:
if it works on both, it should be only in base class